docs/sources/configuration/_index.md

```diff
@@ -196,6 +196,11 @@ configures the HTTP and gRPC server communication of the launched service(s).
 # CLI flag: -server.http-listen-port
 [http_listen_port: <int> | default = 80]
 
+# TLS configuration for serving over https
+[http_tls_config: &tls_server_config]
+  [cert_file: <string>]
+  [key_file: <string>]
+
```

Comment:
Since this refers to the TLS config from weaveworks: 
https://github.com/weaveworks/common/blob/master/server/server.go#L54-L59
and flags:
https://github.com/weaveworks/common/blob/master/server/server.go#L127-L130

It would be nice to describe all possible configuration parameters:
```suggestion
# TLS configuration for serving over HTTPS
http_tls_config:
  # HTTP server cert path.
  # CLI flag: -server.http-tls-cert-path
  [cert_file: <string> | default = ""]

  # HTTP server key path.
  # CLI flag: -server.http-tls-key-path
  [key_file: <string> | default = ""]

  # HTTP TLS Client Auth type.
  # CLI flag: -server.http-tls-client-auth
  [client_auth_type: <string> | default = ""]

  # HTTP TLS Client CA path.
  # CLI flag: -server.http-tls-ca-path
  [client_ca_file: <string> | default = ""]
  
```

Reply:
Is it OK if I suggest to include the same parameters from **grpc_tls_config**?

Reply:
@andretadeu yes it would be great to also include the grpc TLS configs  🎉 
https://github.com/weaveworks/common/blob/master/server/server.go#L131-L134

Reply:
I recreated the PR here: (https://github.com/grafana/loki/pull/6933)